tests/util/helpers.sh

```diff
@@ -15,6 +15,38 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# _set_kube_vars initializes the following variables based on the value of KUBECONFIG:
+#
+# KUBECONFIG_FILES: an array containing the individual files in the order specified.
+# KUBE_CONTEXTS: the names of the kube contexts, in the order of the KUBECONFIG files specified.
+function _set_kube_vars()
+{
+  # Split out the kube config files and then get the first context in
+  # each. We do this because the contexts are stored in a map, which
+  # means that order of the context returned by
+  # `kubectl config get-contexts` is not guaranteed. By pulling out
+  # the context on a per-file basis, we maintain the order of the
+  # files in the KUBECONFIG variable. It should be noted, however,
+  # that this logic assumes one context per file.
```

Comment:
Which logic assumes this? Is it not possible to run the tests using one KUBECONFIG containing multiple contexts/clusters?